libmozdata/lando.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from enum import Enum
+import requests
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class LandoWarnings(object):
+    """
+    Encapsulates lando warning calls
+    """
+
+    class ReturnCode(Enum):
+        OK = 0,
+        ERROR = 1
+
+    def __init__(self, api_url, api_key):
+        self.api_url = "{api_url}/diff_warnings".format(api_url=api_url)
+        self.api_key = api_key
+
+    def del_warnings(self, warnings):
+        """
+        Deletes warnings from Lando based on a json warnings list
+        """
+        for warning in warnings:
+            warning_id = warning["id"]
+
+            response = requests.delete(
+                "{url}/{id}".format(url=self.api_url, id=warning_id),
+                headers={"X-Phabricator-API-Key": self.api_key},
+            )
+
+            if response.status_code != 200:
+                logger.info(
+                    "Failed to delete warning for warning_id {}!".format(warning_id)
+                )
+                return LandoWarnings.ReturnCode.ERROR
```

Comment:
Instead of returning 0 in case of success and 1 in case of error, let's raise an exception when there is an error and don't return anything in case the function is successful.

Reply:
Why would we want exceptions?

Reply:
It's the more Pythonic way to handle errors

Reply:
Cool!